scripts/login_screen.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import tkinter as tk
 import json
-from tkinter import *
+from tkinter import PhotoImage
```

Comment:
For consistency, either keep this import with calls to `PhotoImage` or remove it completely with the calls to `tk.PhotoImage`